source/etherpad.rst

```diff
@@ -0,0 +1,194 @@
+.. highlight:: console
+
+.. sidebar:: About
+
+  This guide has been tested with Uberspace 7.1.1 and Etherpad Lite 1.6.3
+
+#############
+Etherpad Lite
+#############
+
+Etherpad Lite is a real-time collaborative writing tool. It is based on NodeJs and comes with lot of possible plugins.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * Node.js_ and its package manager npm_ 
+  * MySQL_ 
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [etherpad@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [etherpad@stardust ~]$ 
+
+You'll need your MySQL_ credentials. Get them with ``my_print_defaults``:
+
+::
+
+ [etherpad@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=etherpad
+ --password=MySuperSecretPassword
+ [etherpad@stardust ~]$ 
+
+Your domain needs to be setup:
+
+::
+
+ [etherpad@stardust ~]$ uberspace web domain list
+ etherpad.uber.space
+ [etherpad@stardust ~]$ 
+
+Installation
+============
+
+First get the Etherpad Lite source code from Github_:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
+
+Then run the etherpad script, to install the dependencies:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ ~/etherpad/bin/installDeps.sh
```

Comment:
Same here:

```
[etherpad@stardust ~]$ ~/etherpad/bin/installDeps.sh
Copy the settings template to settings.json...
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
[...]
Clearing minified cache...
Ensure custom css/js files are created...
[etherpad@stardust ~]$ 
```

You can always shorten unimportant output with `[...]` like I did here.